style.css

```diff
@@ -0,0 +1,269 @@
+header {
+    background: #373c3f;
+    height: 100px;
+    font-weight: bold;    
+}
+
+body {
+    margin: 0px;
+}
+
+a {
+    color: white;
+    text-decoration: none;
+}
+
+nav {
```

Comment:
I'd highly recommend you to use classes instead of tags in case of browser performance improvement and styles compatibility issues with other libs